wsmaster/che-core-api-factory/src/main/java/org/eclipse/che/api/factory/server/DefaultFactoryParameterResolver.java

```diff
@@ -29,6 +30,10 @@
 @Singleton
 public class DefaultFactoryParameterResolver implements FactoryParametersResolver {
 
+  public static final String LOCAL_FILES_REFERENCES_ARE_NOT_SUPPORTED =
+      "Devfile requires local file content but this functionality is not supported when URL to raw content is specified. "
+          + "You can specify URL to Github repository instead.";
```

Comment:
Is github going to be the only repo we're going to ever support?

Reply:
Currently, only Github is supported. Later any git hosting service may be implemented but it should provide access to files content without cloning it (like https://raw.githubusercontent.com).